### ISSUE TITLE
[Feature Request] Open a specific file

### ISSUE THREAD

**Issue by eval-exec** [paulirish/git-open#198]
Thanks for your `git-open` plugin, it helped me a lot. And I think it would be helpful to add open a file feature, for example:

```bash
$ ls
. .. .git src README.md

$ git open --file README.md # visit "https://github.com/[user]/[repo]/README.md" file on github
```

**Comment by skypesky** [comment]
done in https://github.com/skypesky/to-where-cli/pull/45

**Comment by jonhattan** [comment]
Done in #170

**Comment by NightMachinery** [comment]
emacs `git-link` also supports this.

**Comment by paulirish** [comment]
Closed by #170
